package.json

```diff
@@ -2,7 +2,7 @@
   "name": "esp-idf-extension",
   "preview": true,
   "displayName": "Espressif IDF",
-  "description": "Espressif IoT Development Framework",
+  "description": "Espressif's official vscode extension for developing IoT application using ESP-IDF, aiming ESP-32, ESP-32-S2 chips",
```

Comment:
Product names are "ESP32" and "ESP32-S2" (no hyphen after "ESP")

Also, it seems that the search result tiles in marketplace show only the first ~80 characters of the description. Let's make this description fit. Maybe something like

> Develop and debug applications for Espressif ESP32, ESP32-S2 chips with ESP-IDF

(79 characters)